Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
I'm looking at the `FormContractorInterface` right now and `getFormBuilder` is implemented more or less the same way every time. `getDefaultOptions` has some differences.

- https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/master/Builder/FormContractor.php
- https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/master/Builder/FormContractor.php
- https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/master/Builder/FormContractor.php